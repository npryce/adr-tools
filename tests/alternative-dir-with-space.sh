mkdir dir\ with\ space
cd dir\ with\ space
adr init alt-dir-with-space

ls .
ls alt-dir-with-space

adr list
