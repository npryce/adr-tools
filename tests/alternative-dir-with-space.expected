mkdir dir\ with\ space
cd dir\ with\ space
adr init alt-dir-with-space
alt-dir-with-space/0001-record-architecture-decisions.md

ls .
alt-dir-with-space
ls alt-dir-with-space
0001-record-architecture-decisions.md

adr list
alt-dir-with-space/0001-record-architecture-decisions.md
